src/components/ModalDrawer/ModalDrawer.vue

```diff
@@ -38,7 +38,6 @@
 const props = defineProps<{ title: string; isOpen?: boolean }>();
 const el = ref(null as null | HTMLDivElement);
 const closeButton = ref(null as null | HTMLButtonElement);
-let keydownEventListener = null;
```

Comment:
Looks like this variable is used down there in the code, though not properly, so removing it probably is not what should be done. I guess let's unsubscribe (removeEventListener) using this saved reference instead of setting it to null for now reason.